packages/helix/build.sh

```diff
@@ -3,6 +3,7 @@ TERMUX_PKG_DESCRIPTION="A post-modern modal text editor written in rust"
 TERMUX_PKG_LICENSE="MPL-2.0"
 TERMUX_PKG_MAINTAINER="Aditya Alok <alok@termux.org>"
 TERMUX_PKG_VERSION="24.03"
+TERMUX_PKG_REVISION=1
 TERMUX_PKG_SRCURL="git+https://github.com/helix-editor/helix"
```

Comment:
Moreso a general packaging question.
Why are we building this package via a git clone instead of a release tarball?
There doesn't seem to be anything in the build that would obviously necessitate it.

Sure the script shim for `hx` is `$PREFIX/bin` is a bit unusual, but not related to the actual build process.
(and the actual build should maybe be moved into `termux_step_make()` but if it works it works)

Reply:
> Why are we building this package via a git clone instead of a release tarball?

It was like that since first addition of helix https://github.com/termux/termux-packages/commit/c41c0f0e53dcdf30ea4b5b0db5c49f76b15d7b11

The source can be changed to tarball. That topic should be discussed in another pull request.

Reply:
Gentoo uses source tarball https://gitweb.gentoo.org/repo/gentoo.git/tree/app-editors/helix/helix-24.03.ebuild#n284